src/ui-kit/icons/svg-avatars/index.js

```diff
@@ -0,0 +1,27 @@
+import { ReactComponent as Avatar1 } from './Avatar1.svg';
```

Comment:
Please make normal icons or remove it. And it is better to do import/export in the icons folder, where we already have one icon object